.github/workflows/update-docs-in-website.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: |
           mkdir -p ./pages/docs/tools/generator
           cp -R ../generator/docs ./pages/docs/tools/generator
+          rm ../generator/docs/README.md
+          rm ../generator/docs/jsdoc2md-handlebars
```

Comment:
@Florence-Njeri did you test it on local, you sure it is not `rm -r` or something else?

`rm ../generator/docs/README.md` is fine as you remove one file. But removal of entire folder probably requires some flag

Reply:
@Florence-Njeri hey, please have a look

Reply:
Hey @derberg, sorry I missed your comment. Let me take a look

Reply:
I've tested the `rmdir` command on localhost and it does remove the entire folfer